src/psyclone/psyir/symbols/symboltable.py

```diff
@@ -638,6 +626,24 @@ def get_unresolved_datasymbols(self, ignore_precision=False):
             unresolved_datasymbols = unresolved_symbols
         return [sym.name for sym in unresolved_datasymbols]
 
+    @property
+    def symbols_dict(self):
+        '''
+        :returns:  Ordered dictionary of symbols indexed by symbol name.
```

Comment:
Picky but you only need once space after ":returns:" and below.

Reply:
There were quite a few of these in the file so I've changed them all. I also made all the sentences start with a lower case word.